docs/source/conf.py

```diff
@@ -139,6 +146,11 @@
         },
     ],
     "secondary_sidebar_items": ["page-toc", "source-links", "example-links"],
+    "switcher": {
+        "json_url": switcher_url,
+        "version_match": current[0],
+    },
+    "navbar_start": ["navbar-logo", "version-switcher"],
```

Comment:
To consider:
Changing conf.py in the future would lead to different doc styles (some versions will use the updated conf.py, others -- the old one).
We should overwrite older conf.py with the newest one.

Reply:
Made this into an option at `docs/source/builder.py`, variable `NEW_SPHINX_CONFIGS`.